.github/workflows/release.yml

```diff
@@ -175,6 +176,17 @@ jobs:
           type=semver,suffix=${{ matrix.suffix }},pattern={{major}}.{{minor}}
           type=semver,suffix=${{ matrix.suffix }},pattern={{major}}
 
+    - uses: aws-actions/configure-aws-credentials@v4
+      if: ${{ startsWith(github.ref, 'refs/tags/v') }}
+      with:
+        aws-region: us-east-1
+        role-to-assume: arn:aws:iam::${{ secrets.RP_AWS_CRED_ACCOUNT_ID }}:role/${{ vars.RP_AWS_CRED_BASE_ROLE_NAME }}${{ github.event.repository.name }}
+
```

Comment:
```suggestion
```
i don't think `aws-actions/configure-aws-credentials@v4` is needed here because the exact same role is already assumed in line 136

Reply:
if the region changes, credentials need to be reconfigured